mavros/src/plugins/setpoint_attitude.cpp

```diff
@@ -215,6 +237,43 @@ class SetpointAttitudePlugin : public plugin::PluginBase,
 			send_attitude_ang_velocity(req->header.stamp, ang_vel, thrust_msg->thrust);
 	}
 
+	void rpyt_cb(const mav_msgs::RollPitchYawrateThrustConstPtr msg)
+	{
+		if(ignore_rpyt_messages_){
+			ROS_FATAL("Recieved roll_pitch_yaw_thrust_rate message, but ignore_rpyt_messages_ is true: "
+	 	        "the most likely cause of this is a failure to specify the thrust_scaling_factor,	"
+			"yaw_rate_scaling_factor or system_mass_kg parameters");
+		     	return;
+		}
+
+		// Set mask to ignore everything but thrust and attitude setpoint
+		uint8_t ignore_all_except_q_and_thrust = (7 << 0);
+
+		geometry_msgs::Quaternion orientation = 
+			tf::createQuaternionMsgFromRollPitchYaw( msg->roll, msg->pitch, 0);
```

Comment:
We have `ftf::quaternion_from_rpy()` in `frame_tf.h`.

Reply:
Why Yaw is ignored?

Reply:
The idea is that we can apply a body rate to yaw, as usually the yaw reference can be different than the one onboard (for intance, if we use an external controller and just need to send an actuation to the UAV), while the roll/pitch are the same.